hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/management/event/DistributionTagDropEvent.java

```diff
@@ -127,28 +127,25 @@ private boolean validateIfSourceIsDs(final Table source) {
         return true;
     }
 
+    @SuppressWarnings("unchecked")
```

Comment:
I would move the `@SuppressWarnings("unchecked")` to line `136` because this is the online unchecked operation in the method